### DISCUSSION TITLE
Setting note titles when using folder templates

### DISCUSSION THREAD

**Discussion by EpicPilgrim** [SilentVoid13/Templater#1110] [Help] [answered]
Hi all

I'm using Folder Templates. When I create a new note in a folder with this configured, everything populates as expected. However, is it possible to template the title of the note created? By default, Obsidian just creates it as "Untitled", however depending on the folder, I'd like to customise this some to include date and maybe other words.

Thanks!

**Comment by Zachatoo** [comment]
Sure, in your template you can rename the file to whatever you want

```js
<% await tp.file.rename("MyNewName") %>
```

**Reply by Zachatoo** [reply, marked as answer]
Example using date as part of name
```js
<% await tp.file.rename(tp.date.now("YYYY-MM-DD") + " MyNewName") %>
```

**Reply by EpicPilgrim** [reply]
Perfect! Thanks for that
